src/ansible_compat/runtime.py

```diff
@@ -380,18 +395,21 @@ def _prepare_ansible_paths(self) -> None:
             (roles_path, "roles", True),
         ]
 
-        if self.isolated:
-            alterations_list.extend(
-                [
-                    (roles_path, f"{self.cache_dir}/roles", False),
-                    (library_paths, f"{self.cache_dir}/modules", False),
-                    (collections_path, f"{self.cache_dir}/collections", False),
-                ]
-            )
+        alterations_list.extend(
+            [
+                (roles_path, f"{self.cache_dir}/roles", False),
+                (library_paths, f"{self.cache_dir}/modules", False),
+                (collections_path, f"{self.cache_dir}/collections", False),
+            ]
+            if self.isolated
+            else []
+        )
```

Comment:
Is there a reason for this non-functional change? As of now, this will only make `git blame` users jump through another hoop before they get to the proper result.

Reply:
Yes there is and it will make you laugh. Previous code was failing with C901 as function reached a cyclomatic complexity of 11. Rewriting that condition using the inline condition, tricket flake8 to believe complexity remained the same.

I personally prefer to add `# noqa: C901` to those methods/functions that need extra complexity but I am almost sure that @webknjaz would have requested a change if I went that approach.